elasticsearch/1.9/README.md

```diff
@@ -0,0 +1,99 @@
+# How to use Elasticsearch on DC/OS
+
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a distributed, RESTful search and analytics engine.
+The [DC/OS Elasticsearch](https://github.com/mesos/elasticsearch) service is a Mesos framework that allows you to manage
+and use Elasticsearch in a flexible and scalable way.
+
+- Estimated time for completion: 5 minutes
+- Target audience: Data engineers and appops that want an search engine.
+- Scope: Install and use Elasticsearch.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Elasticsearch](#install-elasticsearch)
+- [Use Elasticsearch](#use-elasticsearch)
+- [Uninstall Elasticsearch](#uninstall-elasticsearch)
+
+## Prerequisites
+
+- A running DC/OS 1.9 cluster with 3 agents with each 1 CPU, 3 GB of RAM and 1 GB of disk space available.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed.
+
+## Install Elasticsearch
+
+To install Elasticsearch, do:
+
+```bash
+$ dcos package install elasticsearch
+The ElasticSearch DCOS Service implementation is alpha and there may be bugs, incomplete features, incorrect documentation or other discrepancies.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [elasticsearch] version [1.0.1-2]
+Elasticsearch is staging. Check Marathon and the ES management front-end for status.
+```
+
+After this, you should see the Elasticsearch service running via the `Services` tab of the DC/OS UI:
+
+![Elasticsearch DC/OS service](img/services.png)
+
+## Use Elasticsearch
+
+In the following we will use the DC/OS [Admin Router](https://dcos.io/docs/1.9/development/dcos-integration/#-a-name-adminrouter-a-admin-router) to provide access to the Elasticsearch UI: use the URL `http://$DCOS_DASHBOARD/service/elasticsearch/` and replace `$DCOS_DASHBOARD` with the URL of your DC/OS UI:
```

Comment:
The Admin Router link is a 404, looks like the docs have moved here for 1.9:
https://dcos.io/docs/1.9/developing-services/#-a-name-adminrouter-a-admin-router-and-web-interface-integration